gensim/test/test_word2vec.py

```diff
@@ -763,6 +763,15 @@ def testLineSentenceWorksWithNormalFile(self):
                 sentences = word2vec.LineSentence(fin)
                 for words in sentences:
                     self.assertEqual(words, utils.to_unicode(orig.readline()).split())
+
+    def testLineSentencePath(self):
+        """Does LineSentencePath work with a path argument?"""
+        logging.debug(word2vec)
+        with utils.smart_open(datapath('lee_background.cor')) as orig:
+            sentences = word2vec.LineSentencePath(datapath('LineSentencePath'))
```

Comment:
Perhaps rather than creating a custom new split (and duplication) of the `lee_background.cor`, a new subdir of test-data `lee` could be added, with just the `lee.cor` and `lee_background.cor` files. The test would make sure this new class on the directory yields the same set of docs as the *two* other files read individually. And eventually, the duplication of these files in two places could be eliminated by making all the other tests/demos/tutorials just grab the lee* files from this new canonical place. (That is: aim for less duplication in the long run by re-using the same files everywhere.)